README.md

```diff
@@ -1,64 +1,316 @@
+    - In **Admin consent display name** type `Access Webapi as a user`
+    - In **Admin consent description** type `Allow the application to access Webapi on behalf of the signed-in user.`
+    - In **User consent display name** type `Access Webapi as a user`
+    - In **User consent description** type `Allow the application to access Webapi on your behalf.`
+    - Keep **State** as **Enabled**
+    - Select **Add scope**
+
+#### Configure the **msal-obo-sample** to use your Azure AD tenant
+
+Open `application.properties` in the src/main/resources folder. Fill in with your tenant and app registration information noted in the above registration step.
+
+- Replace *Enter_the_Tenant_Info_Here* with  Directory (tenant) ID.
+- *Enter_the_Application_Id_here* with the Application (client) ID.
+- *Enter_the_Client_Secret_Here* with the *key value* noted.
+
+#### Register the client Web app (Webapp-Openidconnect)
+
+1. In the **Azure Active Directory** pane, click on **App registrations** and choose **New registration**.
+1. Enter a friendly name for the application, for example 'java-webapp', select **Accounts in any organizational directory and personal Microsoft Accounts (e.g. Skype, Xbox, Outlook.com)**.
+1. Click **Register** to register the application.
+1. On the **Overview** page:
+    - copy **Application (client) ID**
+    - copy **Directory (tenant) ID**
+    - You'll need both of these values later to configure the projects
+1. Select the **Authentication** blade, and under *Redirect URIs*, select "Web".
+    You will need to enter two different redirect URIs: one for the sign-In page, and one for the page that calls Graph. For both, you should use the same host and port number, then followed by "/msal4jsample/secure/aad" for the sign in page and "msal4jsample/graph/me" for the user info page.
+    To run this sample, enter the following two urls :
+
+    - `http://localhost:8080/msal4jsample/secure/aad`.
+    - `http://localhost:8080/msal4jsample/graph/me`
+
+Click on **save**.
+
+1. Select **Certificates & Secrets** blade and click on `New client secret` in the **Client Secrets** section:
+
+   - Type a key description (of instance `app secret`),
+   - Select one of the provided key durations as per your security needs.
+   - When you save this page, the key value will be displayed. Copy and save the value in a safe location. You'll need this key later to configure the project. This key value will not be displayed again, nor retrievable by any other means, so record it as soon as it is visible in the Azure portal.
+
+1. Select **API Permissions** blade:
+   - Click the **Add a permission** button,
+   - Select the **My APIs** tab,
+   - In the list of APIs, select the API `Web Api app`,
+   - In the **Delegated permissions** section, ensure that the right permissions are checked: **access_as_user**,
+   - Select the **Add permissions** button.
+
+#### Configure the **msal-web-sample** to use your Azure AD tenant
```

Comment:
Missing step to add KnownClientApplication on the web api manifest. You can read about it here: https://github.com/Azure-Samples/ms-identity-aspnet-webapi-onbehalfof#configure-known-client-applications-for-service-todolistservice-obo-sample-v2

Reply:
I think that step is not necessary for this Java sample. We are not adding anything in the manifest

Reply:
You could check with @kalyankrishna1 but I think this is mandatory for the first consent screen.

Reply:
added KnownClientApplication step

Reply:
@ramya25 I couldn't find the step to edit it in the manifest

Reply:
Sorry. I think Readme file in the last push is not updated. now updated with the missing steps